_posts/2024-06-03-storybook-visual-testing.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: post
+title: Visual testing with Storybook
+author: juholei
+excerpt: Turn your Storybook stories into visual tests with Storybook Test Runner
+tags:
+  - Frontend
+  - Storybook
+  - Testing
+  - Automation
+  - Test Automation
+
+---
+
+[Storybook](https://storybook.js.org/) is a tool for frontend development which allows you to provide interactive
```

Comment:
for the uninitiated it would be beneficial to know what "frontend" tech it works with? react? or can I use it with server side frontend stuff as well.

Reply:
Added clarification